contributing/topics/guide-resource-ids.md

```diff
@@ -0,0 +1,109 @@
+# Guide: Resource IDs
+
+Resource IDs are an essential component of all resources and data sources within the provider and are required in order to interact with the corresponding API and to be tracked in state correctly by Terraform.
+
+Due to their fundamental importance in the provider, resource IDs can and should be handled through the use of various helper functions that are available.
```

Comment:
I'd drop can here, since we're unlikely to accept any submission that is using IDs directly and would require handlers to be added / updated to deal with this?
```suggestion
Due to their fundamental importance in the provider, resource IDs should be handled through the use of various helper functions that are available.
```